compare-git-hashes.sh

```diff
@@ -22,41 +22,42 @@ clean_checkout() {
 		git fetch https://github.com/stan-dev/cmdstan +refs/pull/$prNumber/merge:refs/remotes/origin/pr/$prNumber/merge
         git checkout refs/remotes/origin/pr/$prNumber/merge
 	else
-		git checkout "$1"
+		git checkout "$1" && git pull origin "$1"
 	fi
+    git clean -xffd
```

Comment:
Sorry to be nitpicky but can you fix indentation? let's unify with spaces instead of tabs and looks like most of the file is 4 space indent